bin/docker-entrypoint

```diff
@@ -11,8 +11,8 @@ else
     echo "No tables exist - setting up the database"
     rails db:setup
   else
-    echo "Running the migrations"
-    rails db:migrate
+    echo "Running the schema & data migrations"
+    rails db:migrate:with_data
```

Comment:
Looking at the definition for this rake task makes me wary. It looks like it re-implements `db:migrate`. Unless there's good reason, I'd feel more comfortable running `db:migrate data:migrate` separately.

Reply:
👍